cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -120,7 +150,8 @@ class ScalarFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
-    section("id()", "functions-id") {
+    section("[deprecated]#id()", "functions-id") {
```

Comment:
This delta is missing in the https://development.neo4j.dev/docs/cypher-manual/5.0/deprecations-additions-removals-compatibility/